app/models/petition.rb

```diff
@@ -876,8 +876,7 @@ def publish!(time = Time.current)
 
     update!(
       state: state_for_publishing(time),
-      open_at: time_for_publishing(time),
-      closed_at: closing_date(time)
```

Comment:
Might want to investigate where else this method may be used - it might be possible to remove it